pandas/core/reshape/concat.py

```diff
@@ -20,7 +20,7 @@
 
 def concat(objs, axis=0, join='outer', join_axes=None, ignore_index=False,
            keys=None, levels=None, names=None, verify_integrity=False,
-           copy=True):
```

Comment:
actually move before verify_integrity

Reply:
pls do this

Reply:
Why make this an API breaking change?

Reply:
because it more logical.

Reply:
How so?

I'm OK with breaking API when necessary, but this seems unnecessary.